src/Cache/Context/OgRoleCacheContext.php

```diff
@@ -67,16 +87,22 @@ public static function getLabel() {
    *
    * @param \Drupal\Core\Session\AccountInterface $user
    *   The current user.
-   * @param \Drupal\og\MembershipManagerInterface $membership_manager
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   The entity type manager.
+   * @param \Drupal\og\MembershipManagerInterface $membershipManager
    *   The membership manager service.
-   * @param \Drupal\Core\PrivateKey $private_key
+   * @param \Drupal\Core\Database\Connection $database
+   *   The active database connection.
+   * @param \Drupal\Core\PrivateKey $privateKey
    *   The private key service.
    */
-  public function __construct(AccountInterface $user, MembershipManagerInterface $membership_manager, PrivateKey $private_key) {
+  public function __construct(AccountInterface $user, EntityTypeManagerInterface $entityTypeManager, MembershipManagerInterface $membershipManager, Connection $database, PrivateKey $privateKey) {
```

Comment:
I'm OK with using camel case vs snake case here but I think it would be the first time we use camel case for the formal parameters in a constructor.

Reply:
Oh I was not even aware any more of this change. Both are allowed according to the coding standards, and there is no preference. I don't think it is important to be consistent in this. I have a personal preference for the camelcased variant since I think it is more modern and in line with other PHP frameworks, but this is just personal and I'm willing to adapt for OG.

Reply:
> I don't think it is very important for OG to be consistent in this.
Except when there are other PR's that also add/inject new items, you'll get awkward merge issues.

> I have a personal preference for the camelcased variant since I think it is more modern and in line with other PHP frameworks.
I agree with this.

Since there's probably no other PR changing this class I'm fine with this.